docs/source/workflow_inputs.rst

```diff
@@ -578,7 +578,7 @@ Building occupancy is entered in ``/HPXML/Building/BuildingDetails/BuildingSumma
   =======================================================  ========  =====  ===========  ========  ========  ========================
 
   .. [#] If NumberofResidents not provided, an *asset* calculation is performed assuming standard occupancy, in which occupant-driven end uses (e.g., plug loads, appliances, hot water) are calculated based on NumberofBedrooms.
-         If NumberofResidents is provided, an *operational* calculation is instead performed, in which hot water end uses are calculated based on NumberofResidents from `Estimating Daily Domestic Hot-Water Use in North American Homes <http://www.fsec.ucf.edu/en/publications/pdf/fsec-pf-464-15.pdf>`_ and all other occupant-driven end uses are adjusted using the following relationship from `RECS 2015 <https://www.eia.gov/consumption/residential/reports/2015/overview/>`_:
+         If NumberofResidents is provided, an *operational* calculation is instead performed, in which *some* occupant-driven end uses, as described throughout the documentation, are adjusted using the following relationship from `RECS 2015 <https://www.eia.gov/consumption/residential/data/2015/>`_:
```

Comment:
Is there a plan to transition to RECS 2020 for number of bedrooms?

Reply:
Yes, @prsh5175 was working on that [here](https://github.com/NREL/OpenStudio-HPXML/pull/1690), but we ran into issues with hot water use. Thus we ended up pursuing an [alternative hot water use approach](https://github.com/NREL/OpenStudio-HPXML/pull/1761). Assuming testing looks good and it gets merged, we should be able to revisit and wrap up Prateek's PR after that.